src/remote/config.go

```diff
@@ -99,17 +99,17 @@ func (a *StorageApi) GetConfigRequest(branchId int, componentId string, configId
 
 // CreateConfigRequest https://keboola.docs.apiary.io/#reference/components-and-configurations/component-configurations/create-development-branch-configuration
 func (a *StorageApi) CreateConfigRequest(config *model.ConfigWithRows) (*client.Request, error) {
-	// Id is autogenerated
```

Comment:
Povodne slo vytvarat iba configy bez definovaneho ID, pouzivalo sa to v testoch.
Teraz je mozne vytvorit aj config, ktory ma ID vygenerovane dopred, s pouzitim `persist` prikazu.